credentialdigger/scanners/git_scanner.py

```diff
@@ -72,7 +73,7 @@ def scan(self, git_url, since_commit=None, max_depth=1000000):
         repo = GitRepo(project_path)
 
         try:
```

Comment:
is this try-except block still needed to know if a repo is empty?
Can we use the `branches` variable such that we don't add useless operations?